Mathlib/GroupTheory/QuotientGroup/Basic.lean

```diff
@@ -281,6 +286,14 @@ noncomputable def quotientInfEquivProdNormalQuotient (H N : Subgroup G) [N.Norma
   (quotientMulEquivOfEq (by simp [φ, ← comap_ker])).trans
     (quotientKerEquivOfSurjective φ φ_surjective)
 
+/-- **Noether's second isomorphism theorem**: given two subgroups `H` and `N` of a group `G`,
+where `N` is normal, defines an isomorphism between `H/(H ∩ N)` and `(HN)/N`. -/
+@[to_additive "Noether's second isomorphism theorem: given two subgroups `H` and `N` of a group `G`,
+where `N` is normal, defines an isomorphism between `H/(H ∩ N)` and `(H + N)/N`"]
+noncomputable def quotientInfEquivProdNormalQuotient (H N : Subgroup G) [hN : N.Normal] :
+    H ⧸ N.subgroupOf H ≃* _ ⧸ N.subgroupOf (H ⊔ N) :=
```

Comment:
I think adding `H ⊔ N` might help with readability (though the original statement did also omit the `H ⊔ N`)
```suggestion
    H ⧸ N.subgroupOf H ≃*  (H ⊔ N : Subgroup G) ⧸ N.subgroupOf (H ⊔ N) :=
```